modules/network-observability-dns-overview.adoc

```diff
@@ -13,10 +13,18 @@ You can configure graphical representation of Domain Name System (DNS) tracking
 
 * Troubleshooting: Debug DNS-related issues by tracing DNS resolution steps, tracking latency, and identifying misconfigurations.
 
-When DNS tracking is enabled, you can see the following metrics represented in a chart in the *Overview*. See the _Additional Resources_ in this section for more information about enabling and working with this view.
+When DNS tracking is enabled, you can see the following non-empty metrics represented in a donut or line chart in the *Overview* by default.
```

Comment:
Is "donut" the official name? I don't see it elsewhere in the docs.

Reply:
Stating _in the *Overview* by default_ almost sounds like you can configure things to show the metrics elsewhere.
```suggestion
By default, when DNS tracking is enabled, you can see the following non-empty metrics represented in a donut or line chart in the *Overview* page.
```

Reply:
Hmm I can see what you mean. I was trying to convey that the mentioned metrics are visible by default. What I'm not saying is that there are others you can configure to view that aren't shown by default...I will adjust the wording here to clarify that.